Source/SwiftLintFramework/Protocols/Rule.swift

```diff
@@ -13,12 +13,35 @@ public protocol Rule {
     func validateFile(file: File) -> [StyleViolation]
 }
 
+extension Rule {
```

Comment:
Is it not possible to make `Rule` conform to `Equatable` and implement `==`? Would be a bit more Swifty than `isEqualTo`. Or `ParameterizedRule` for that matter.